README.md

```diff
@@ -5,23 +5,64 @@ Prometheus is an Open source monitoring tool.  A BOSH release was created for it
 The BOSH release deploys the following monitoring tools - [Prometheus](<https://prometheus.io/>), [AlertManager](https://github.com/prometheus/alertmanager) and  [Grafana](<https://grafana.com/>).
 
 
-## Usage
+# Usage
```

Comment:
I'm going to be THAT guy - there's actually a WCAG accessibility thing that a page should only contain one h1